projects/core/components/data-list/option/option.component.ts

```diff
@@ -97,6 +97,7 @@ export class TuiOptionComponent<T = unknown> implements OnDestroy {
     onClick(): void {
         if (this.host && this.value !== undefined) {
             this.host.handleOption(this.value);
+            this.el.nativeElement.focus();
```

Comment:
Why do we need it? It gets focused on hover, what is the case where it can be clicked while not focused?